pnpm-lock.yaml

```diff
@@ -8278,7 +8295,7 @@ packages:
     resolution: {integrity: sha512-oUwtPJ1W0SKD0Tr+wqu92c5xuCeQqB3hSCHasn/ZgjFdA9iDGNkNf2Zi9ztY7X+hNuMib23LNGRm6+uN+KLE3g==}
     engines: {node: '>=8.10.0'}
     peerDependencies:
-      eslint: '*'
+      eslint: '>=5.16.0 || *'
```

Comment:
Update your pnpm version. You are not using the latest v6, that is why the lockfile gets changed like this.